### ISSUE TITLE
Notice: offset in logged timestamps

### ISSUE THREAD

**Issue by negoguy** [kernc/logkeys#180]
I've noticed that there is an offset in the timestamps logged in the output file. 

This seems to be caused because the next event's timestamp is written in the output file right after an event's happening, even if the next event is to happen several minutes/hours/days later.

```
Logging started ...

2018-02-16 20:24:21+0000 > Logging started ...

2018-02-20 19:08:12+0000 > 03458263 <-scanned the 20
2018-02-20 19:08:29+0000 > 03458263 <-scanned the 20
2018-02-20 19:08:32+0000 > 03458263 <-scanned the 21 but timestamp written the 20 
2018-02-21 18:16:50+0000 > 03458263 <-scanned the 21
2018-02-21 18:17:24+0000 > <-timestamp already written in file, even if the event haven't happened yet
```

This is not an actual issue nor a source of great imprecision (if we are aware of it)  but i still think this could be adressed easely (at first glance). 

Best regards

**Comment by kernc** [comment]
Yeah, the timestamp is [printed when <kbd>Enter</kbd> key is pressed](https://github.com/kernc/logkeys/blob/7a9f19fb6b152d9f00a0b3fe29ab266ff1f88129/src/logkeys.cc#L575-L585).
